source/stylesheets/base/_layout.scss

```
@@ -11,3 +11,11 @@ a {
 	}
 }
 
+.layout {
+	max-width: 60em;
+	@include margin($spacing-base);
+
+    @include breakpoint($breakpoint-medium) {
+		@include margin($spacing-large auto);
```

No need to use Bourbon’s margin mixin here. In this case, the output of the mixin is the same as just using vanilla CSS:

```suggestion
margin: $spacing-large auto;
```